jetty-core/jetty-websocket/jetty-websocket-jetty-server/src/main/java/org/eclipse/jetty/websocket/server/WebSocketUpgradeHandler.java

```diff
@@ -117,11 +123,16 @@ public WebSocketUpgradeHandler configure(Consumer<ServerWebSocketContainer> conf
     @Override
     public boolean handle(Request request, Response response, Callback callback) throws Exception
     {
-        if (_container.handle(request, response, callback))
+        if (handle(_container, request, response, callback))
```

Comment:
oh it is so much nicer to review code that has underscores in from of field names!